eng/pipelines/runtimelab/install-emscripten.ps1

```diff
@@ -0,0 +1,23 @@
+param(
+    [string]$InstallDir
+)
+
+New-Item -ItemType Directory -Force -ErrorAction SilentlyContinue -Path (Split-Path -Path $InstallDir -Parent) -Name (Split-Path -Path $InstallDir -Leaf)
```

Comment:
Does this really need to be this elaborate?

I just tried:
```
/home/singleaccretion> New-Item /home/singleaccretion/test/xyz/dir -Force -ItemType Directory
```
And it created the nested directory as expected.

Reply:
Its ok on Linux, but with a drive letter it fails.  We could strip the drive letter and assume we are "in" that drive, but it probably isn't much easier to read.
```
PS C:\Users\scott> new-item -ItemType Directory -Force -Name c:/tmp/1/2
new-item : The given path's format is not supported.
At line:1 char:1
+ new-item -ItemType Directory -Force -Name c:/tmp/1/2
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [New-Item], NotSupportedException
    + FullyQualifiedErrorId : System.NotSupportedException,Microsoft.PowerShell.Commands.NewItemCommand

PS C:\Users\scott> new-item -ItemType Directory -Force -Name /tmp/1/2


    Directory: C:\Users\scott\tmp\1


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d-----        11/05/2024     16:54                2
```

Reply:
> `new-item -ItemType Directory -Force -Name c:/tmp/1/2`

I think this should use `-Path`. I tried:
```
PS C:\Users\Accretion\AppData\Local\Temp> new-item -Force -ItemKind Directory -Path C:\Users\Accretion\AppData\Local\Temp/xyz/rot/tot

    Directory: C:\Users\Accretion\AppData\Local\Temp\xyz\rot

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d----          13.05.2024    00:22                tot
```